daemon/export_test.go

```diff
@@ -203,6 +204,19 @@ func MockSnapstateInstallPathMany(f func(context.Context, *state.State, []*snap.
 	}
 }
 
+func MockRebootAndCaptureArgs(rebootAction *boot.RebootAction,
+	durationUntilReboot *time.Duration, rebootInfo *boot.RebootInfo) func() {
+
+	mockReboot := func(ra boot.RebootAction, d time.Duration, ri *boot.RebootInfo) error {
+		*rebootAction = ra
+		*durationUntilReboot = d
+		rebootInfo = ri
```

Comment:
this doesn't really capture anything, you would need a ** (pointer to pointer) to capture but see the comment about the approach

Reply:
it seems this is not noticeable because there's no test where rebootInfo is set to something? one of the existing tests in daemon for RebootReboot should probably grow passing a real rebootInfo